src/main/java/org/spectrumauctions/sats/core/model/mrvm/MRVMLocalBidder.java

```diff
@@ -18,8 +23,10 @@
  *
  */
 public final class MRVMLocalBidder extends MRVMBidder {
+	
+	private boolean allowAssigningLicensesWithZeroBasevalueInDemandQuery = false;
```

Comment:
I'm a bit concerned about this field.
Why do we have it (it is neither a reasonable constant nor can it be changed)? Given that your implementation can both handle `true`and `false` nicely, do you want to allow changing the behavior at runtime? Given that there seem to be no advantages from setting this to `true`, I might prefer not to use this field at all and just always apply your modifications, i.e., remove line 107 `if(!this.allowAssigningLicensesWithZeroBasevalueInDemandQuery) {`.

Do you understand and agree with my thoughts? Don't hesitate to correct me if I got something wrong.

Reply:
Note: The same applies to the other class where you have an identical field as well, of course...

Reply:
This flag is there to allow backwards compatibility. At the moment there is no need to expose this to users but this can be added later (see GSVM legacy version for an example). Therefore I would leave it as it is at the moment.

Reply:
I see. Then again, if we want backwards compatibility, wouldn't we have to give the user a way to set this to `true`?
Anyways... This is not yet going into master but into feat-gsvm-v2 (which I known nothing about :-)) and I'll leave the choice to you and @islerfab  :-)

Reply:
I agree with both of your arguments. There is a good reason to have this flag and its logic, and I think think it should be exposed to the user, such that she can reproduce old results. I added it to our (still a bit clumsy, that's an open refactoring task) builder / setup classes. If you don't agree at all, @beyeler , feel free to correct me.